src/platform/site-wide/hooks/static-drupal-data.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-unused-vars */
+
+import environment from 'platform/utilities/environment';
+import { useEffect, useState } from 'react';
+
+// UseEffect to fetch data async, starts as an [] and then goes to data array (fake for now)
+// For now immediately returns a resolved promise
+// Test to make sure an empty array still renders properly
+// Set this up as a reusable custom hook? Wrapper around useEffect code and all the calls. Check docs, sync with Claudio on this?
+
+// Use environment.API_URL for getting the API endpoint right for the environment
+
+const fakeSupplementalCOVIDStatus = [
+  {
+    name: 'COVID-19 health protection: Levels low',
+    description:
+      '\u003Cul\u003E\r\n\t\u003Cli\u003EEveryone must wear a mask.\u003C/li\u003E\r\n\t\u003Cli\u003EVisitors are welcome.\u003C/li\u003E\r\n\t\u003Cli\u003EPlease self-screen. If you\u0027ve had any COVID-19 symptoms in the past 48 hours, call first unless you need care now.\u003Cbr /\u003E\r\n\t\u003Ca href=\u0022https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html\u0022\u003ECheck COVID-19 symptoms on the CDC website\u003C/a\u003E\u003C/li\u003E\r\n\u003C/ul\u003E\r\n\r\n\u003Cp\u003ECheck on Mondays for each facility\u2019s latest guidelines.\u003C/p\u003E\r\n',
+    status_id: 'COVID_LOW',
+  },
+  {
+    name: 'COVID-19 health protection: Levels high',
+    description:
+      '\u003Cul\u003E\r\n\t\u003Cli\u003EEveryone must wear a mask and practice physical distancing.\u003C/li\u003E\r\n\t\u003Cli\u003EWe may screen you for COVID-19 symptoms at the entrance.\u003C/li\u003E\r\n\t\u003Cli\u003EApproved visitors only. Ask your care team who can visit.\u003C/li\u003E\r\n\t\u003Cli\u003EAsk your team about phone or video appointment options.\u003C/li\u003E\r\n\u003C/ul\u003E\r\n\r\n\u003Cp\u003ECheck on Mondays for each facility\u2019s latest guidelines.\u003C/p\u003E\r\n',
+    status_id: 'COVID_HIGH',
+  },
+  {
+    name: 'COVID-19 health protection: Levels medium',
+    description:
+      '\u003Cul\u003E\r\n\t\u003Cli\u003EEveryone must wear a mask.\u003C/li\u003E\r\n\t\u003Cli\u003EYour care partner is welcome (no\u0026nbsp;other visitors).\u003C/li\u003E\r\n\t\u003Cli\u003EPlease self-screen. If you\u0027ve had any COVID-19 symptoms in the past 48 hours, call first unless you need care now.\u003Cbr /\u003E\r\n\t\u003Ca href=\u0022https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html\u0022\u003ECheck COVID-19 symptoms on the CDC website\u003C/a\u003E\u003C/li\u003E\r\n\u003C/ul\u003E\r\n\r\n\u003Cp\u003ECheck on Mondays for each facility\u2019s latest guidelines.\u003C/p\u003E\r\n',
+    status_id: 'COVID_MEDIUM',
+  },
+];
+
+export default function useStaticDrupalData(url) {
+  const [staticDrupalData, setStaticDrupalData] = useState([]);
+
+  useEffect(() => {
+    const fetchPromise = fetch(url);
+
+    fetchPromise.then(response => {
+      setStaticDrupalData(response);
+    });
+  }, []);
```

Comment:
Lint warning here - just pass in the `url` variable into the `useEffect` 2nd arg array - `[url]` - since the effect function is dependent on that value being provided.